helper-scripts/backup_and_restore.sh

```diff
@@ -222,7 +222,7 @@ elif [[ ${1} == "restore" ]]; then
 
   echo "[ 0 ] all"
   # find all files in folder with .tar.gz extension, print their base names, remove backup_, remove .tar, remove .gz
```

Comment:
You might want to change the comment to :)

Reply:
Sorry for the delay ... of course the comment should be changed too. 
I created a commit for this :)